### ISSUE TITLE
[Suggestion] Read only system wide containers

### ISSUE THREAD

**Issue by dnkmmr69420** [89luca89/distrobox#578]
There should be system wide read only containers that can be ran by any user without root. But it should require root if you want to install packages on the container. This would be very useful on silverblue so you can install system wide packages for all users without layering any packages. It would also be nice so non admin users can run the containers but not modify the containers without root privileges. Maybe there can be an option where you create a root container, you set an option to make it available for all users but it will be read only unless you enter the container with root privileges.

**Comment by luc14n0** [comment]
Hi there,

From a **Distrobox** user to another. Going through the hoops to make that work reliably doesn't seem worth it IMHO, if _possible_ at all.

Basically, **Distrobox** would have to implement a way to migrate _rootful_ containers to _rootless_ containers, and vice-versa, and taking into account any quirks of the distros it want to support - in this case, pretty much all of them. And bear in mind that a container can't be _rootful_ and _rootless_ at the same time. So, having those restrictions on normal users not being able to modify the container might not even be possible.

To sum it up, if **Podman** or **Docker** can't do it, **Distrobox** won't either.

**Comment by dnkmmr69420** [comment]
I am gonna bring this up to podman

**Comment by dnkmmr69420** [comment]
I thought of an alternative idea. Distrobox can have rootless containers that will automaticly be coppied to other users. Like there is the main container and the coppied containers. a user with sudo privleges can type sudo distrobox copy-to-other-users then the containers will be coppied to all other users.

**Comment by luc14n0** [comment]
This looks more achievable than the original preposition, at least at first sight.

**Comment by 89luca89** [comment]
Due to the nature of podman and docker, this is not possible
And copying the same container over-and-over for root access is not a good solution

So I'll mark this as a wontfix
